src/macros.rs

```diff
@@ -283,6 +291,38 @@ macro_rules! intrinsics {
         intrinsics!($($rest)*);
     );
 
+    // Explicit weak linkage gets dropped when weak-intrinsics is on since it
+    // will be added unconditionally to all intrinsics and would conflict
+    // otherwise.
+    (
+        #[linkage = "weak"]
```

Comment:
This is not doing what I want it to do, and I am still getting double linkage errors. Will debug, surely I don't understand something about macro_rules

Reply:
Right, I understand and will need to make some changes to how this works in order to avoid adding conditionals to every linkage="weak" elsewhere.

Reply:
Got it now.

Without weak-intrinsics:
00000000 W __aeabi_memclr8   // Is explicitly weak
00000000 T __udivsi3  // Is not.

With weak-intrinsics:
00000000 W __aeabi_memclr8
00000000 W __udivsi3